cpp/daal/src/data_management/finiteness_checker.cpp

```diff
@@ -247,6 +247,8 @@ double computeSumSOAAVX512Impl(NumericTable & table, bool & sumIsFinite)
     daal::threader_for_break(nCols, nCols, [&](size_t i, bool & needBreak) {
         double * localSum     = tlsSum.local();
         bool * localNotFinite = tlsNotFinite.local();
+        DAAL_CHECK_MALLOC_THR(localSum);
```

Comment:
You do not use or check the status `safeStat` after the end of the for. You need to add status to the function `computeSumSOAAVX512Impl` - or return

Reply:
Fixed